accounts/forms.py

```diff
@@ -1,9 +1,19 @@
 from django import forms
+from django.forms import ModelForm
 from django.contrib.auth.forms import UserCreationForm
 from django.contrib.auth.models import User
 
+from .models import Profile
+
 
 class CreateUserForm(UserCreationForm):
     class Meta:
         model = User
         fields = ["username", "email", "password1", "password2"]
+
+
+class ProfileForm(ModelForm):
+    class Meta:
+        model = Profile
+        fields = ["first_name", "last_name", "email", "profile_pic", "phone"]
+        exclude = ['user']
```

Comment:
skoro podałeś jakie fieldy mają być, to ten exclude wydaje się niepotrzebny, zobacz czy jak usuniesz to będzie działało tak samo